src/util/defaults.ml

```diff
@@ -78,7 +78,7 @@ let _ = ()
       ; reg Std "solverdiffs"     "false"        "Print out solver differences."
       ; reg Std "allfuns"         "false"        "Analyzes all the functions (not just beginning from main). This requires exp.earlyglobs!"
       ; reg Std "nonstatic"       "false"        "Analyzes all non-static functions."
-      ; reg Std "colors"          "false"        "Colored output."
+      ; reg Std "colors"          "'auto'"       "Colored output (via ANSI escape codes). 'auto': enabled if stdout is a terminal (instead of a pipe); 'always', 'never'."
```

Comment:
In the scripts, we should probably just remove the `--enable colors` argument then.

https://github.com/goblint/analyzer/blob/75a14da737510a406ecc10ef93a2462a6ca97e98/regtest.sh#L12

https://github.com/goblint/analyzer/blob/75a14da737510a406ecc10ef93a2462a6ca97e98/scripts/dgoblint#L11-L12

https://github.com/goblint/analyzer/blob/75a14da737510a406ecc10ef93a2462a6ca97e98/scripts/spec/check.sh#L18

(I think the last one also needs more updates since `--sets result html` is gone as of #108 )

Reply:
We can probably remove `scripts/dgoblint` or does someone use it?

Reply:
Removed 06bcba2